data/json/mapgen_palettes/military/mil_base_palette.json

```diff
@@ -95,7 +95,7 @@
       "M": "f_makeshift_bed",
       "C": "f_crate_c",
       "L": "f_shower",
-      "6": "f_water_heater",
+      "6": [ "f_water_heater", "f_water_heater_family" ],
```

Comment:
i guess military facility can benefit from even heavier heaters, and shouldn't alternate between them

Reply:
That’s reasonable.